qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -159,6 +159,8 @@ def main(self,argsOverride=None,bitcoinConfDict=None,wallets=None):
         random.seed(self.randomseed)
         print("Random seed: %s" % self.randomseed)
 
+        self.options.tmpdir += '/' + str(self.options.port_seed)
```

Comment:
@awemany you are right `os.path.join(..) ` is definitely better and a portale way to handle file path